src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -1,97 +1,118 @@
 package edu.kis.powp.jobs2d;
 
-import java.awt.EventQueue;
-import java.awt.event.ActionEvent;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
 import edu.kis.legacy.drawer.panel.DefaultDrawerFrame;
 import edu.kis.legacy.drawer.panel.DrawPanelController;
 import edu.kis.powp.appbase.Application;
-import edu.kis.powp.jobs2d.drivers.adapter.MyAdapter;
+import edu.kis.powp.jobs2d.drivers.adapter.Job2DToDrawPanelAdapter;
+import edu.kis.powp.jobs2d.drivers.command.DriverCommandFactory;
 import edu.kis.powp.jobs2d.events.SelectChangeVisibleOptionListener;
 import edu.kis.powp.jobs2d.events.SelectTestFigureOptionListener;
 import edu.kis.powp.jobs2d.features.DrawerFeature;
 import edu.kis.powp.jobs2d.features.DriverFeature;
 
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 public class TestJobs2dPatterns {
+
 	private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
 	/**
 	 * Setup test concerning preset figures in context.
-	 * 
+	 *
 	 * @param application Application context.
 	 */
 	private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
 
-		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest(SelectTestFigureOptionListener.FIGURE_JOE_1,
+				selectTestFigureOptionListener);
+		application.addTest(SelectTestFigureOptionListener.FIGURE_JOE_2,
+				selectTestFigureOptionListener);
+		application.addTest("Create Square", e -> {
+			DriverCommandFactory.createSquare().execute(
+					DriverFeature.getDriverManager().getCurrentDriver()
+			);
+		});
+		application.addTest("Create Big Square", e -> {
+			DriverCommandFactory.createBigSquare().execute(
+					DriverFeature.getDriverManager().getCurrentDriver()
+			);
+		});
+		application.addTest("Create Triangle", e -> {
+			DriverCommandFactory.createTriangle().execute(
+					DriverFeature.getDriverManager().getCurrentDriver()
+			);
+		});
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)